test/WebJobs.Extensions.Tests/Extensions/NotificationHubs/NotificationHubConfigurationTests.cs

```diff
@@ -97,12 +91,29 @@ public void Resolve_UsesDefault_Last()
             Assert.Equal("Default", connString);
         }
 
-        private NotificationHubsConfiguration InitializeConfig(string defaultConnStr)
+        [Fact]
```

Comment:
I'd like to see a few more tests - maybe setting HubName on attributes, both attribute & config, existing and non-existing app settings, %% signs and no percent signs.